internal/build/lifecycle_execution_test.go

```diff
@@ -714,6 +714,48 @@ func testLifecycleExecution(t *testing.T, when spec.G, it spec.S) {
 				})
 			})
 		})
+
+		when("override GID", func() {
+			when("override GID is provided", func() {
+				it("configures the phase with the expected arguments", func() {
+					verboseLifecycle := newTestLifecycleExec(t, true, func(options *build.LifecycleOptions) {
+						options.GID = 0
+					})
+					fakePhaseFactory := fakes.NewFakePhaseFactory()
+
+					err := verboseLifecycle.Create(context.Background(), false, "", true, "test", "test", "test", fakeBuildCache, fakeLaunchCache, []string{}, []string{}, fakePhaseFactory)
+					h.AssertNil(t, err)
+
+					lastCallIndex := len(fakePhaseFactory.NewCalledWithProvider) - 1
+					h.AssertNotEq(t, lastCallIndex, -1)
+
+					configProvider := fakePhaseFactory.NewCalledWithProvider[lastCallIndex]
+					h.AssertEq(t, configProvider.Name(), "creator")
+					h.AssertIncludeAllExpectedPatterns(t,
+						configProvider.ContainerConfig().Cmd,
+						[]string{"-gid"},
+					)
+				})
+			})
+			when("override GID is not provided", func() {
```

Comment:
I'm a bit confused by the differences between the previous test and this. Both seem to be doing the same thing, so what's happening here?

Reply:
Oh! you are right, but it wasn't the intention, let me fix it, the idea was to send a negative GID and verify the flag -gid was not added. My mistake after removing the boolean attribute